src/schema.rs

```diff
@@ -18,10 +24,12 @@ enum SortField {
     CompressedSize,
     UncompressedSize,
     CompressionRatio,
+    NullCount,
 }
 
 #[component]
-pub fn SchemaSection(parquet_info: super::DisplayInfo) -> impl IntoView {
+pub fn SchemaSection(parquet_reader: super::ParquetReader) -> impl IntoView {
```

Comment:
I think the `ParquetReader` is now deprecated, instead, we should use the `ParquetTable` which provides a similar interface.